From c95c36473d1469e022cfc510289b25162b84ac2e Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Tue, 5 Jan 2021 13:37:13 +0100
Subject: [PATCH 0083/1141] dt-bindings: stm32: add st,stm32mp25-syscfg
 compatible for syscon

Add the new syscon compatible for STM32MP25 syscfg = "st,stm32mp25-syscfg".

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Change-Id: Id3a67d11e0ec4d4faa4ca40c0a1f35b729a399a4
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/189712
---
 Documentation/devicetree/bindings/arm/stm32/st,stm32-syscon.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/stm32/st,stm32-syscon.yaml b/Documentation/devicetree/bindings/arm/stm32/st,stm32-syscon.yaml
index 6f846d69c5e1..fb71d574b5a5 100644
--- a/Documentation/devicetree/bindings/arm/stm32/st,stm32-syscon.yaml
+++ b/Documentation/devicetree/bindings/arm/stm32/st,stm32-syscon.yaml
@@ -16,6 +16,7 @@ properties:
       - items:
           - enum:
               - st,stm32mp157-syscfg
+              - st,stm32mp25-syscfg
               - st,stm32mp151-pwr-mcu
               - st,stm32-syscfg
               - st,stm32-power-config
-- 
2.39.2

